decidim-proposals/app/controllers/concerns/decidim/proposals/orderable.rb

```diff
@@ -28,7 +28,7 @@ def available_orders
         end
 
         def default_order
-          if current_settings.votes_blocked?
+          if votes_visible?
```

Comment:
I don't really know how the `votes_hidden` option relates to `votes_blocked`, but the change seems good because this way we're sure the option is available since the `available_orders` method does the exact same check. 

I guess we could even assume the "most_voted" option is enabled in this case and not go through `detect_order` at all. Just do:

```diff
         def default_order
-          if current_settings.votes_blocked?
-            detect_order("most_voted")
+          if votes_visible?
+            "most_voted"
           else
             "random"
           end
```